lib/cc/cli/init.rb

```diff
@@ -9,6 +9,7 @@ class Init < Command
 
       def run
         if !upgrade? && filesystem.exist?(CODECLIMATE_YAML)
+          create_default_configs
           fatal "Config file .codeclimate.yml already present.\nTry running 'validate-config' to check configuration."
```

Comment:
@wfleming cool. Thanks for the explanation. I'd like consistently generating a successful exit code.

What was our original reason in favor of going with the `exitcode 1` when there's no `.codeclimate.yml`? What would you think about something more neutral like `.codeclimate.yml already present, skipping generation`, like we do for each engine config that's already present?